app/connectors/SoftDrinksIndustryLevyConnector.scala

```diff
@@ -83,38 +83,30 @@ class SoftDrinksIndustryLevyConnector @Inject()(
   def balance(
                sdilRef: String,
                withAssessment: Boolean
-             )(implicit hc: HeaderCarrier): VariationResult[BigDecimal] = EitherT {
```

Comment:
The return types of these have been changed. Are they used by anywhere else in the service?

Reply:
No these are only used this pages so far